composer.json

```diff
@@ -10,13 +10,13 @@
   "require": {
     "php": ">=7.3",
     "ext-json": "*",
-    "code-foundation/flow-config": "^0.1.0",
     "doctrine/orm": "^2.6.3",
     "eoneopay/apiformats": "dev-master",
     "eoneopay/externals": "^1.0@rc",
     "eoneopay/framework": "dev-master",
     "eoneopay/utils": "^0.2.8",
     "loyaltycorp/requesthandlers": "^1.1",
+    "loyaltycorp/flowconfig-wrapper": "dev-master",
```

Comment:
Dependency should be above `loyaltycorp/requesthandlers` (alpha order)

Reply:
Fixed!